Docs/Migration-INI.md

```diff
@@ -0,0 +1,854 @@
+# Migrating from older versions - INI configuration
+This guide uses [YR mod base](https://github.com/Starkku/cncnet-client-mod-base) configuration as an example. The majority of changes also applies to to non-YR client configurations.
+
+It is **highly recommended** to make a complete backup of your mod before starting.
+
+## Edit `ClientDefinitions.ini`
+1. Add `[Settings]->UnixLauncherExe=Launcher.sh` (script file name can be anything)
+2. Create `Launcher.sh` in root directory:
+```sh
+#!/bin/sh
+
+cd "$(dirname "$0")"
+dotnet Resources/Binaries/UniversalGL/clientogl.dll "$@"
+```
+3. Add these entries in `[Settings]` (fill with your required/forbidden mod files):
+```ini
+; Comma-separated list of files required to run the game / mod that are not included in the installation.
+RequiredFiles=
+; Comma-separated list of files that cannot be present to run the game / mod without problems.
+ForbiddenFiles=
+```
+
+## Edit `SkirmishLobby.ini`
+1. Change `[INISystem]->BasedOn` from `MultiplayerGameLobby.ini` to `GameLobbyBase.ini`.
+2. Add `$BaseSection=GameLobbyBase` to `[SkirmishLobby]`
+3. Remove `[lbMapList]` section
+4. Add these two sections:
+```ini
+[GameOptionsPanel]
+$CC-GODD03=cmbGameSpeedCapSkirmish:GameLobbyDropDown
+
+[cmbGameSpeedCapSkirmish]
+$BaseSection=cmbGameSpeedCap
+Items=Fastest (MAX),Faster (60 FPS),Fast (30 FPS),Medium (20 FPS),Slow (15 FPS),Slower (12 FPS),Slowest (10 FPS)
+```
+
+## Add `GameLobbyBase.ini`
+
+This file is the base layout of all game lobbies (skirmish, LAN, CnCNet). **Game options have been moved from `GameOptions.ini` to this file**.
+To add controls in the game options panel, add `$CC-GO` prefixed list entries in `[GameOptionsPanel]`, then create their own sections.
+See example configuration below.
+
+```ini
+[INISystem]
+BasedOn=GenericWindow.ini

```

Comment:
This big code block asks to be spoilered.

Reply:
Also can you add an example on how to port one example checkbox or dropdown? something like "My cool game option" before and after migration, with all the relevant code. I think that would be more useful than the example config because the example configs could be found in the new mod base and/or TSCv6 which we should mention instead.